project/frontend/src/FindMatch/FindMatch.js

```diff
@@ -0,0 +1,45 @@
+import React from "react"
+import ReactDOM from "react-dom"
+
+class PreferenceForm extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = { cuisine: "" }
```

Comment:
I think you could probably store all the fields in the state (i.e. priceLevel, distance, etc.)? I know this PR is just for one input but just thinking for how it can work with all the inputs

Reply:
And then for your changeState function you could pass in another parameter or something indicating which part of the state to change... something like (not sure if this exactly would work but you get the idea)
`changeState(event, key) {
  this.setState({key: event.target.value})
}`

Reply:
yeah that's what's going to happen!